.github/workflows/ci.yml

```diff
@@ -43,6 +43,23 @@ jobs:
                 curl -L -O -k https://gitlab.tiker.net/inducer/ci-support/raw/main/build-and-test-py-project.sh
                 . ./build-and-test-py-project.sh
 
+    pytest_macos:
+        name: Pytest on Py${{ matrix.python-version }} (macOS)
+        runs-on: macos-latest
+        strategy:
+            matrix:
+                python-version: ['3.7', '3.x']
+        steps:
+        -   uses: actions/checkout@v2
+        -
+            uses: actions/setup-python@v1
+            with:
+                python-version: ${{ matrix.python-version }}
+        -   name: "Main Script"
+            run: |
+                curl -L -O -k https://gitlab.tiker.net/inducer/ci-support/raw/main/build-and-test-py-project.sh
+                . ./build-and-test-py-project.sh
```

Comment:
@inducer Does this need to do anything special for macOS? I don't think I've ever set up a CI for that :fearful:

Reply:
Ought to just work. E.g.:

https://github.com/inducer/pyopencl/blob/1c31380430d1c262bfe12014fa62e95ae55d0902/.github/workflows/ci.yml#L76-L88=